debugtools/DDR_VM/data/superset-constants.dat

```diff
@@ -1576,7 +1576,7 @@ C|J9_EXTENDED_RUNTIME_RESTRICT_IFA
 C|J9_EXTENDED_RUNTIME_SET_HW_PREFETCH
 C|J9_EXTENDED_RUNTIME_UNUSED_0x40000000
 C|J9_EXTENDED_RUNTIME_UNUSED_0x800000
-C|J9_EXTENDED_RUNTIME_USER_REALTIME_ACCESS_BARRIER
+C|J9_EXTENDED_RUNTIME_USER_SATB_ACCESS_BARRIER
```

Comment:
No additions should be made to this file. Each constant listed here should have always been defined in source code.

Anything not used in code under `debugtools/DDR_VM/src` can safely be deleted. If new constants need to be used in DDR code, they should be declared (with a default value) in `CompatibilityConstants29.dat`.

Reply:
_Just to clarify my understanding:_
- if new constants aren't used in DDR (i.e., under `debugtools/DDR_VM/src`) they don't need to be declared in `CompatibilityConstants29.dat`, is this correct?
- Anything exisiting in `superset-constants.dat` that isn't used in DDR can also be removed

Reply:
That is correct.

BTW, last week I worked on changes to remove unnecessary entries from `superset-constants.dat` (an entry there doesn't make it usable from DDR code - it must also be declared in a header file). My plan is to deliver those changes shortly.

Reply:
Thanks!

To summarize, the entries in question for this PR are:
```
C|OMR_GC_WRITE_BARRIER_TYPE_REALTIME
C|gc_modron_wrtbar_realtime
C|j9gc_modron_wrtbar_realtime
C|J9_EXTENDED_RUNTIME_USER_REALTIME_ACCESS_BARRIER
C|J9VMCONSTANTPOOL_COMIBMOTIVMVM_J9_GC_WRITE_BARRIER_TYPE_REALTIME
```

These are no longer referenced in code as this PR renames the consts, plus none of these were used by DDR to begin with. So I will remove them from the file and we won't need to bother adding the new ones to `CompatibilityConstants29.dat`